doc/classes/OS.xml

```diff
@@ -970,7 +970,18 @@
 				Shows the virtual keyboard if the platform has one.
 				The [code]existing_text[/code] parameter is useful for implementing your own [LineEdit] or [TextEdit], as it tells the virtual keyboard what text has already been typed (the virtual keyboard uses it for auto-correct and predictions).
 				The [code]multiline[/code] parameter needs to be set to [code]true[/code] to be able to enter multiple lines of text, as in [TextEdit].
-				[b]Note:[/b] This method is implemented on Android, iOS and UWP.
+				[b]Note:[/b] This method is implemented on Android, iOS, UWP, and HTML5.
```

Comment:
Might be worth adding a line to explain the relationship between `[method show_virtual_keyboard]` and `[method show_virtual_keyboard_type]`, e.g.:
```suggestion
				[b]Note:[/b] This method is equivalent to calling [method show_virtual_keyboard_type] with either default or multiline keyboard type. It is kept for compatibility with previous Godot releases and should be considered [i]deprecated[/i] and replaced by [method show_virtual_keyboard_type].
				[b]Note:[/b] This method is implemented on Android, iOS, UWP, and HTML5.
```